add: new patterns to pollution (#132)
aphp/edsnlp: changelog.md

- Add attribute `section` to entities
- Add new cases for separator pattern when components of the TNM score are separated by a forward slash
- Add NER `eds.adicap` pipeline to identify ADICAP codes

> - Add pattenrs to `pollution` pipeline and simplifies activate/deactivate of specific patterns

Author: ```suggestion
- Add patterns to `pollution` pipeline and simplifies activate/deactivate of specific patterns
```

> - Add patterns to `pollution` pipeline and simplifies activate/deactivate of specific patterns

Reviewer: ```suggestion
- Add patterns to `pollution` pipeline and simplifies activating or deactivating specific patterns
```

> ### Changed

Reviewer: A word about the breaking change ?

```suggestion
### Changed
- Simplified the configuration scheme of the `pollution` pipeline
```
